skfda/exploratory/visualization/_outliergram.py

```diff
@@ -50,22 +63,39 @@ class Outliergram:
     def __init__(
         self,
         fdata: FDataGrid,
+        chart: Union[Figure, Axes, None] = None,
+        *,
+        fig: Optional[Figure] = None,
+        axes: Optional[Axes] = None,
+        n_rows: Optional[int] = None,
+        n_cols: Optional[int] = None,
+        **kwargs,
     ) -> None:
+        BasePlot.__init__(self)
         self.fdata = fdata
         self.depth = ModifiedBandDepth()
         self.depth.fit(fdata)
         self.mbd = self.depth(fdata)
         self.mei = self.modified_epigraph_index_list()
+        if self.mbd.size != self.mei.size:
+            raise ValueError(
+                "The size of mbd and mei should be the same.",
+            )
+        self.n = self.mbd.size
+        distances, parable = self._compute_distances()
+        self.distances = distances
+        mei_ordered = self.mei[:]
+        mei_ordered, parable = (
```

Comment:
Use `np.argsort` to find the ordered indexes. It should be more efficient.

Reply:
but i dont want the indices, i want to order the parable array as the mei, so that i can plot it in order

Reply:
Once you have the indices you can use them to index both arrays.

Reply:
ok true, i was testing it wrongly